SU-SOE/stanford_soe_helper
# SOE-2076: Fixed the padding below the menu

# READY FOR REVIEW

# Summary
- This fixes the padding below the main menu. 

# Needed By (Date)
- 

# Criticality
- 

# Steps to Test
- Visit the magazine. Make sure that on all pages within them menu the space below the menu is at 15px.
 -- /magazine
 -- /magazine/issue/29
 -- /magazine/article/how-do-people-relearn-walk-after-surgery
 -- /magazine/articles
 -- /magazine/issue
- Visit other pages on the site _outside of the magazine_, say the homepage, and verify the spacing around the masthead matches that on [Prod](https://engineering.stanford.edu/).


# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2076
## Related PRs

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
